packages/docs/src/index.md

```diff
@@ -1,15 +1,6 @@
----
-home: true
-heroText: SCALE
-actionText: Get Started
-actionLink: /guide/introduction
-features:
-  - title: Type-Safe
-    details: API is strongly typed in the most cases. Something is partially up to you for performance reasons.
-  - title: Performant
-    details: Benchmarked and optimizied to encode and decode data in the most efficient way (but maybe it could be even more efficient!).
-  - title: Modular
-    details: Use higher-level codec abstractions or low-level core library to build such abstraction on your own.
-  - title: Debuggable
-    details: Higher-level library provides a tracking functionality to find a way what goes wrong in decoding process
----
+# Scale JS
+
+This project aims to be a comprehensive solution to handle [SCALE Codec](https://docs.substrate.io/reference/scale-codec/) in JavaScript with great speed and first-class TypeScript support.
+
+See [API reference](./api/).
```

Comment:
this link is broken

Reply:
It is not )
But it will be visible only after the merging.